### PR TITLE
Update icon and styling

### PR DESCRIPTION
**Pull request by GRA0007**
I've updated the icons used to a proper square icon, and adjusted some styling so the buttons in the popup match Wanikani more closely and the text is more streamlined.

### CODE REVIEW

File: options.js [hunk 1]
```diff
@@ -17,26 +17,26 @@ function save_options() {
...
 
-// retrieve current status and updates checkbok
-function update_checkbox() {
-    chrome.storage.sync.get("notifyWKlessons", function(result) {
-        var lessonsenabled = result.notifyWKlessons;
-
+// Retrieve options from chrome.storage
+function load_options() {
+    chrome.storage.sync.get(["notifyWKlessons", "WKapikey"], function(result) {
+        document.getElementById('apikey').value = result.WKapikey;
```
Comment (BleckJeck) [inline options.js:31]: I don't really like to unnecessarily expose the API key through the UI.
Can you undo this change?

File: background.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ function check() {
                 })
         }
     })
-    window.setTimeout(check, 10000);
+    window.setTimeout(check, 60000);
```
Comment (BleckJeck) [inline background.js]: From a UX point of view, not sure I like this one. It will lead to a big gap while you're in a review session.
I think 1 call every 10 seconds is very reasonable, but I'm open to discussion.

**Review by BleckJeck — CHANGES_REQUESTED** [review]
Thanks for taking the time to improve the styling, aesthetics is definitely not my strong suit!
Just commented on a few issues I'm not entirely comfortable with.
Happy to merge once these are resolved.

File: options.js (hunk 1 above)
Comment (GRA0007) [reply options.js:31]: So the API is already "exposed" through the WK website, and this settings page is entirely local. Leaving it blank when an API key has already been entered is confusing because it appears that it needs to be entered again. Due to the minute security risks, I believe that the UX is more important here. Let me know if you still disagree though.

File: background.js (hunk 2 above)
Comment (GRA0007) [reply background.js]: This is fair enough. I don't like that the API is checked every 10 seconds, that's a total of 8640 requests a day, which is quite excessive since WK reviews are only available on the hour. I'm happy to revert this change for now, but I would like to see if the extension could check when you finish your reviews on WK and force-check then, as well as every hour. Let me know your thoughts, I'm sure it's possible but I'll have to look into it.

File: options.js (hunk 1 above)
Comment (BleckJeck) [reply options.js:31]: That's fair, I was thinking of a situation where you'd install the extension on a shared computer but I appreciate that's not a very realistic use case now that you mention it, so happy to implement this.

File: background.js (hunk 2 above)
Comment (BleckJeck) [reply background.js]: Totally agree with you here in principle. Technically 10 sec is still within the [api specs](https://docs.api.wanikani.com/20170710/#rate-limit), but still not a smart nor necessary use of server resources. Ideally, the extension should check whether the active tab is either wanikani.com/lesson or wanikani.com/review. If not, as you say, there's no need to check so often.
Shall we write this down as future development and discard the change for now, to maintain the current UX?

**Review by BleckJeck — CHANGES_REQUESTED** [review]

**Comment by BleckJeck** [comment]
Also, feel free to add yourself as a contributor in the options page!

**Comment by GRA0007** [comment]
Oops! Sorry I accidentally marked that comment as resolved. I agree, I'll revert the changes for now and create an issue to track that :)

**Review by BleckJeck — APPROVED** [review]
Thanks - looking good!
I'll merge and submit the changes to the Chrome Web Store over the weekend.
